app/components/speaker-list.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
Once we implement network calls, that data will first originate in `public/index` route. So, we need to make it as close to that as possible.